### ISSUE TITLE
Add missing description of populateFromArray and readOnly.

### ISSUE THREAD

**Issue by aninde** [handsontable/handsontable#8493]
**Context**
WIP

**Description**
Add missing description of the behavior of cells that have the `readOnly` option enabled. They can't be changed by `populateFromArray`, because this method bypasses `cellMeta` with `readOnly`. 
This action is not intuitive for everyone, so it should be explained and described.

**Explanation (example from code of `populateFromArray`)**

```
if (cellMeta.readOnly) {
                current.col += 1;
                /* eslint-disable no-continue */
                continue;
              }
```
**Task to be done**
- [ ] add missing explanation to `populateFromArray`, with a link to `readOnly` option description
- [ ] add missing explanation to `readOnly`, with a link to populateFromArray method and different method to add data e.g. `setDataAtCell`

**Demo**
version 7.1.1 http://jsfiddle.net/aninde/qhot9c71/

**Your environment**
Handsontable version: 7.1.1
Browser Name and version: Google Chrome version 76.0.3809.100 
Operating System: macOS Mojave 10.14.6

**Comment by AMBudnik** [comment]
Still valid for v9 of Handsontable https://jsfiddle.net/tp18967k/ and the new docs.
The `populateFromArray` method does not change the `readOnly` cell, which is not mentioned in the readOnly tutorial at https://handsontable.com/docs/disabled-cells/#overview

**Comment by kirszenbaum** [comment]
Fixed in #9724
